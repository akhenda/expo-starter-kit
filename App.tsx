import { StatusBar } from 'expo-status-bar';

import '@src/config';

import Main from './src';

export default function App() {
  return (
    <>
      <StatusBar style="auto" backgroundColor="transparent" translucent />
      <Main />
    </>
  );
}
