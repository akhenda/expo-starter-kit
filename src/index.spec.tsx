import React from 'react';
import renderer from 'react-test-renderer';

import App from '.';

jest.mock('./hooks/useOnAppStart', () => jest.fn().mockResolvedValue(true));

describe('<App />', () => {
  it('has 1 child', () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
