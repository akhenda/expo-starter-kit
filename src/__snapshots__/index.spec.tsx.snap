// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> has 1 child 1`] = `null`;

exports[`<Entry /> has 1 child 1`] = `null`;
